app/src/main/java/me/hufman/androidautoidrive/carapp/notifications/ReadoutInteractions.kt

```diff
@@ -62,7 +62,13 @@ class ReadoutInteractions(val settings: NotificationSettings) {
 	}
 
 	fun triggerDisplayReadout(carNotification: CarNotification) {
```

Comment:
What is this change for, instead of having the new code call `triggerDisplayReadout` to use the same conditional?

Reply:
agree, that is a bit distracting. I'm changing this adding an additional boolean parameter to triggerDisplayLayout that allows to override the setting. (It shall readout by click on the action and the action shall only be visible if the setting is not checked)

Reply:
done